src/main/java/com/gooddata/executeafm/afm/ExpressionFilter.java

```diff
@@ -27,12 +29,27 @@ public ExpressionFilter(@JsonProperty("value") final String value) {
         this.value = value;
     }
 
-   public String getValue() {
```

Comment:
typo?

Reply:
nope, it's fix of missing space - methods were not aligned :)

Reply:
ohh, sorry, my bad %)